phpBB/phpbb/mention/controller/user_mention.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ *
+ * phpBB mentions. A controller class for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2016, paul999, https://www.phpbbextensions.io
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+	namespace phpbb\mention\controller;
```

Comment:
Please format this file correctly regarding spacing. There should be no spaces/tabs before the namespace, see the coding guidelines.